docs/operator-manual/upgrading/2.4-2.5.md

```diff
@@ -46,6 +46,10 @@ It is **strongly recommended** to leave this check enabled. Disabling the check
 
 The bundled Kustomize version has been upgraded from 4.4.1 to 4.5.5.
 
+## Upgraded Helm Version
+
+Note that bundled Helm version has been upgraded from 3.9.0 to 9.9.3.
```

Comment:
s/9.9.3/3.9.3

Reply:
Fixed,  thanks for the catch.   I thought the coffee was a bit weak this morning.

Reply:
Haha it happens. Time to brew that 2nd batch.